test/integration/test_ssh_access.py

```diff
@@ -0,0 +1,15 @@
+import os
+import pytest
+
+from conftest import find_container
+from exasol_integration_test_docker_environment.lib.base.ssh_access import SshKeyCache
+
+
+def test_generate_ssh_key_file(api_database):
+    params = { "db_os_access": "SSH" }
+    with api_database(additional_parameters=params) as db:
+        cache = SshKeyCache()
+        container = find_container("db_container", db.name)
```

Comment:
the db should contain a environment_info and this should contain a database_info and this a container_info